lib/Basics/win-utils.cpp

```diff
@@ -281,19 +283,21 @@ char* TRI_GETCWD(char* buffer, int maxlen) {
   wchar_t* rcw;
   int wBufLen = maxlen;
   wchar_t* wbuf = (wchar_t*)malloc(wBufLen * sizeof(wchar_t));
```

Comment:
this will leak the memory of `wbuf` if an exception occurs below.

Reply:
Instead of all this `char*` stuff, can't we just make this function return an `std::string`?
There seems to be only one caller of this function in the entire source code.

Reply:
if we use std::wstring for this, we will need to cast away const of c_str()